tests/arch/arm/arm_thread_swap/src/arm_thread_arch.c

```diff
@@ -231,7 +234,7 @@ static void alt_thread_entry(void)
 	 * later, that it is populated properly.
 	 */
 	memset(&ztest_thread_fp_callee_saved_regs,
-		0, sizeof(_callee_saved_t));
+		0, sizeof(struct _preempt_float));
```

Comment:
Why not `sizeof(ztest_thread_fp_callee_saved_regs)` so everybody knows what's going on, and it's robust against changes at the definition?

Reply:
Sure. I've updated that, now, @pabigot 
Thanks